### PR TITLE
qa/vstart_runner: fix the admin_socket args parsing

### PR DESCRIPTION
**Pull request by lxbsz**
Fixes: https://tracker.ceph.com/issues/49621
Signed-off-by: Xiubo Li <xiubli@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: qa/tasks/vstart_runner.py [hunk 1]
```diff
@@ -829,7 +829,7 @@ def admin_socket(self, daemon_type, daemon_id, command, check_status=True,
         if stdout is None:
             stdout = StringIO()
 
-        args=[CEPH_CMD, "daemon", f'{daemon_type}', f'{daemon_id}'] + command
+        args=[CEPH_CMD, "daemon", "{0}.{1}".format(daemon_type, daemon_id)] + command
```
Comment (rishabh-d-dave) [inline qa/tasks/vstart_runner.py:832]: nit:
```suggestion
        args=[CEPH_CMD, "daemon", "{}.{}".format(daemon_type, daemon_id)] + command
```

**Review by rishabh-d-dave — APPROVED** [review]

File: qa/tasks/vstart_runner.py (hunk 1 above)
Comment (lxbsz) [reply qa/tasks/vstart_runner.py]: Done. Thanks.

File: qa/tasks/vstart_runner.py [hunk 2]
```diff
@@ -829,7 +829,7 @@ def admin_socket(self, daemon_type, daemon_id, command, check_status=True,
         if stdout is None:
             stdout = StringIO()
 
-        args=[CEPH_CMD, "daemon", f'{daemon_type}', f'{daemon_id}'] + command
+        args=[CEPH_CMD, "daemon", "{}.{}".format(daemon_type, daemon_id)] + command
```
Comment (batrick) [inline qa/tasks/vstart_runner.py:832]: err, why not

```suggestion
        args=[CEPH_CMD, "daemon", f"{daemon_type}.{daemon_id}"] + command
```

Comment (lxbsz) [reply qa/tasks/vstart_runner.py]: This looks better. Done. Thanks.

**Comment by batrick** [comment]
https://pulpito.ceph.com/pdonnell-2021-03-18_13:46:31-fs-wip-pdonnell-testing-20210318.024145-distro-basic-smithi/

- https://tracker.ceph.com/issues/49466
    qa: "Command failed on gibba030 with status 1: 'set -ex\nsudo dd of=/tmp/tmp.ZEeZBasJer'"
- https://tracker.ceph.com/issues/48773
    qa: scrub does not complete
- https://tracker.ceph.com/issues/48805
    mds: "cluster [WRN] Scrub error on inode 0x1000000039d (/client.0/tmp/blogbench-1.0/src/blogtest_in) see mds.a log and `damage ls` output for details"
- https://tracker.ceph.com/issues/45434
    qa: test_full_fsync (tasks.cephfs.test_full.TestClusterFull) failed
- https://tracker.ceph.com/issues/49845
    qa: failed umount in test_volumes
- https://tracker.ceph.com/issues/49605
    mgr: drops command on the floor
- https://tracker.ceph.com/issues/48203 (stock kernel update required)
    qa: quota failure
- https://tracker.ceph.com/issues/49928
    client: items pinned in cache preventing unmount x2

**Pull request merged**